modules/tinymce/src/core/main/ts/api/util/EventDispatcher.ts

```diff
@@ -228,11 +222,11 @@ class EventDispatcher<T> {
         Tools.extend(wrappedCallback, extra);
       }
 
-      names = name.toLowerCase().split(' ');
-      i = names.length;
+      const names = name.toLowerCase().split(' ');
+      let i = names.length;
       while (i--) {
-        name = names[i];
-        handlers = this.bindings[name];
+        const name = names[i];
```

Comment:
This shadows the name variable passed in the arguments, so this shouldn't be done.